src/util/file-helpers.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+import { type Cleaner } from 'cleaners'
+import { type Disklet } from 'disklet'
+
+/**
+ * Reads, parses, and cleans a JSON file.
+ * Returns undefined if something goes wrong.
+ */
+export async function getFileJson<T>(
```

Comment:
Not that it matters too much, but I think this can "nicer" if it would be written as a "curried" function:

```js
getFileJson<T>(cleaner) => (disklet, path: string) =>  Promise<T | void>
```

That way, you could "wrap" the different cleaners into "explicit" getters like "getStoreItemFile", "getStoreIdFile" etc.

It just seems that you always use the same cleaners from the top scope and they are never really dynamic, so why not incapsulate them. But again, it doesn't really matter :)

Reply:
Ok, let's do it this way:

```js
const storeItemFile = makeJsonFile(asStoreItemFile)

await storeItemFile.load(disklet, path);
await storeItemFile.save(disklet, path, { ...contents })
```

This also addresses the saving side of things, which never really felt clean in the old way of doing it.